src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -17,8 +17,8 @@
 #include <unistd.h>
 
 #include "ray/common/network_util.h"
-#include "ray/protobuf/gcs_service.grpc.pb.h"
 #include "ray/rpc/grpc_client.h"
+#include "src/ray/protobuf/gcs_service.grpc.pb.h"
```

Comment:
why we have "src" prefix here

Reply:
Unfortunately for protobuf headers I couldn't figure out a way to make it work without `src`.